src/app/clusters/thermostat-server/thermostat-server.cpp

```diff
@@ -116,10 +117,10 @@ MatterThermostatClusterServerPreAttributeChangedCallback(const app::ConcreteAttr
     int16_t UnoccupiedCoolingSetpoint;
     int16_t UnoccupiedHeatingSetpoint;
 
-// TODO re-enable reading reaturemap once https://github.com/project-chip/connectedhomeip/pull/9725 is merged
 #ifndef FEATURE_MAP_OVERIDE
-    emberAfReadServerAttribute(endpoint, Thermostat::Id, chip::app::Clusters::Globals::Attributes::Ids::FeatureMap,
-                               (uint8_t *) &FeatureMap, sizeof(FeatureMap));
+    emberAfReadServerAttribute(endpoint, chip::app::Clusters::Thermostat::Id,
```

Comment:
Please use Accessors functions, which have the right types and don't need you to do reinterpret_casting.....

Reply:
This seems old code that was not updated. The constants are all wrong in name.

Tried to improve  on it on #12800 however I could not find the accessor to use even after you mentioned it (before I was not aware of accessors but was thinking "we would need something better").

Could you link to the file to use?

Reply:
Looks like we are not outputting accessors for this!  I will lget that fixed, but in the meantime obviously this PR should not block on that.

Reply:
Will check if there are accessors for FeatureMap.  They were not there so